### ISSUE TITLE
Add working leaderboard

### ISSUE THREAD

**Issue by chrisinajar** [OpenAngelArena/OAA-Public#1]
Use this endpoint: http://chrisinajar.com:6969/top
Those are steam32 ID's

**Comment by IBurn36360** [comment]
Same as #2, is this being worked on?

**Comment by IBurn36360** [comment]
![](https://areqgaming.com/uploads/chrome_2018-01-26_03-20-41.png)

I tossed this up in 10 minutes.  If you give me an idea of what you want it to look like and what functionality you want, I can very easily code it.  These changes are being made in a way that isn't blocking to the responsiveness fixes.

**Comment by vatsalyagoel** [comment]
This looks good to me. follows similar strategy to Dota2 Official leaderboards

**Comment by chrisinajar** [comment]
Pop it in a PR, we can always enhance it more later. Getting something simple up is a billion times better than having nothing. Since I've added names to the API it should be super trivial, that was the only hard part before.

**Comment by IBurn36360** [comment]
I'm getting the Mongo caching layer in before I get the PR up. I'll do a bit of responsive testing to ensure the table doesn't overflow screens as the client width shrinks.  Worst case, I'll make it scroll.  Do we plan on making this just a static asset, or is there any consideration to making it searchable?

**Comment by vatsalyagoel** [comment]
Make it a static asset

**Comment by IBurn36360** [comment]
It needs to update as the leaderboard itself does.  As an additional note, though web dev is my day job, this is my first dive into Angular, so I'm having to do a lot of reading in order to get things in place.

**Comment by vatsalyagoel** [comment]
Hey @IBurn36360 I did a temp version of the leaderboard. Is there anything in particular you want to add to that?

**Comment by IBurn36360** [comment]
Yours still calls to Chris' server from the client side, but it works if we need to get the leaderboard up.  I'd just be ready to pull it in case someone decides to be a troll and hammer Chris' server through the site.

**Comment by vatsalyagoel** [comment]
yeah I'm not exposing chris's server anymore and I have rate limited my own url
